invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/creators.html

```diff
@@ -11,10 +12,12 @@
 <div class="ui accordion">
   <div class="title">
     {{ show_creatibutors(record.ui.creators.creators, show_affiliations=True) }}
+    {% if record.ui.creators.affiliations %}
     <a style="cursor: pointer;" class="dropdown">
```

Comment:
If the below class is needed to make this line work, maybe we should add a class here e.g `affiliation-dropdown` and apply there the rule `cursor: pointer !important; `